rialto_airflow/utils.py

```diff
@@ -49,3 +49,12 @@ def normalize_doi(doi):
     doi = re.sub("^doi: ", "", doi)
 
     return doi
+
+
+def normalize_orcid(orcid):
+    orcid = orcid.strip().lower()
+    orcid = orcid.replace("https://orcid.org/", "").replace(
+        "https://sandbox.orcid.org/", ""
+    )
```

Comment:
sandbox.orcid.org is their test environment -- it is possible we will run into ORCIDs like this if using the ORCID test system (e.g. https://sandbox.orcid.org/0000-0002-7262-6251)